pom.xml

```diff
@@ -931,6 +931,7 @@
           </dependency>
         </dependencies>
         <configuration>
+          <skip>true</skip> <!-- Skip this until we've released Netty 5.0.0.Final -->
```

Comment:
The modified artefact id means revapi cannot find the older version of the modules, and will start reporting ABI problems.